.github/workflows/development-environment.yml

```diff
@@ -81,12 +82,12 @@ jobs:
           restore-keys: |
             ${{ matrix.os }}-yarn-
 
-      # We cannot call make bootstrap directly since run-dependent-services requires some Docker magic.
-      # We have the magic in the getsentry/bootstrap-develop. We will handle this situation in the next pass of this.
-      # make init-config will *not* prompt about overwriting ~/.sentry/config.yml' since we're on a pristine state
+      # make develop calls "ensure-venv", thus, needing to be within a venv
+      # TODO: We need to streamline the installation of the wheel package
       - name: Set up development environment (mostly as per docs)
         run: |
           python -m venv .venv
           source .venv/bin/activate
+          pip install wheel
```

Comment:
This should be moved to like, `install-py-dev` or similar. To resolve https://github.com/getsentry/sentry/issues/24351.